### PR TITLE
DL-1372: tests added

### PR DESCRIPTION
**Pull request by lbobba**

### CODE REVIEW

File: src/test/scala/com/gannett/AppTest.scala [hunk 1]
```diff
@@ -1,17 +1,49 @@
 package com.gannett
```
Comment (DINESHKUMARMURUGAN) [inline src/test/scala/com/gannett/AppTest.scala]: change package to com.gannett.test

File: src/test/scala/GannettSharedSparkContext.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.gannett
```
Comment (DINESHKUMARMURUGAN) [inline src/test/scala/GannettSharedSparkContext.scala]: change package to com.gannett.test

File: src/test/scala/com/gannett/AppTest.scala [hunk 3]
```diff
@@ -1,17 +1,49 @@
...
-@Test
-class AppTest {
 
-    @Test
-    def testOK() = assertTrue(true)
+class AppTest extends FunSuite with GannettSharedSparkContext with ShouldMatchers{
 
-//    @Test
-//    def testKO() = assertTrue(false)
+    test("test App") {
```
Comment (oaustegard) [inline src/test/scala/com/gannett/AppTest.scala]: This is not sufficiently descriptive

File: src/test/scala/com/gannett/AppTest.scala [hunk 4]
```diff
@@ -1,17 +1,49 @@
...
+
+        val df = spark.read
+          .format("com.databricks.spark.csv")
+          .option("header", "true")
+          .schema(customSchema)
+          .load("src/test/resources/input.csv")
+
+        json = Functions.getDiscreteVariableStats(spark, tableName, Column, df)
+        json1 = Functions.getBinaryDiscreteVariableStats(spark, tableName, Column, df)
+        println(json.toJSONString())
```
Comment (oaustegard) [inline src/test/scala/com/gannett/AppTest.scala]: @DINESHKUMARMURUGAN is this all that we can test in terms of descriptive stats? Are these meaningful tests?

**Pull request merged**